graph_defs.go

```diff
@@ -20,3 +22,9 @@ func (c *Client) CreateGraphDefs(graphDefs []*GraphDefsParam) error {
 	_, err := requestPost[any](c, "/api/v0/graph-defs/create", graphDefs)
 	return err
 }
+
+// DeleteGraphDef deletes a graph definition.
+func (c *Client) DeleteGraphDef(name string) error {
+	_, err := requestJSON[any](c, http.MethodDelete, "/api/v0/graph-defs/delete", map[string]string{"name": name})
```

Comment:
Oops, this is the first DELETE API requiring the request body :)

Reply:
Yes :) We did so, although it is somewhat unusual, because while it is common to DELETE a resource by including an ID or similar in the URL path, I was hesitant to include a part of the metric name in the path as an ID.

Reply:
We considered the idea of putting the `name` parameter in the URL path, but we thought it was unnatural as our internal data model (sorry for the internal talk). We are still puzzled, but we decided to put it in the request body. I hope you will forgive me.

Regarding the second topic, you are absolutely right about the `/delete` in the URL path. It should be `DELETE /api/v0/graph-defs`. We will fix it, so please wait.